tests/test_phase_psf.py

```diff
@@ -24,6 +24,11 @@
 import unittest
 from galsim_test_helpers import *
 
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
```

Comment:
Note: I noticed that this pattern wasn't being used in ImSim.  Jim just did `import pickle`.  My understanding is that cPickle is much faster when it is available, so might be worth adding this when you're doing the imsim update.

Reply:
Done on https://github.com/LSSTDESC/imSim/pull/235